### PR TITLE
feat: update ecosystem page

### PR DESCRIPTION
**Pull request by dschlabach**
**What changed? Why?**

**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/2</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>1</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>2</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

File: apps/web/src/components/Ecosystem/Content.tsx
Comment (brendan-defi) [inline apps/web/src/components/Ecosystem/Content.tsx:113]: nit, but stylistically we're moving towards 
```
() => ([
...
])
```

instead of brackets with return statements

**Review by brendan-defi — DISMISSED** [review]

**Comment by brendan-defi** [comment]
@dschlabach you've got to resolve merge conflicts before merging

**Review by alessey — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for alessey @ 2024-12-05 20:51:36 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

**Review by alessey — APPROVED** [review]

**Review by alessey — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for alessey @ 2024-12-05 21:05:21 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

**Comment by cb-heimdall** [comment]
Review Error for alessey @ 2024-12-05 21:06:06 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

**Review by brendan-defi — APPROVED** [review]

**Review by alessey — DISMISSED** [review]

**Comment by cb-heimdall** [comment]
Review Error for brendan-defi @ 2024-12-05 21:07:35 UTC
User failed mfa authentication, see [go/mfa-help](http://go/mfa-help)

**Review by alessey — APPROVED** [review]

**Review by alessey — APPROVED** [review]

**Pull request merged**
